### PR TITLE
Merge #18502: doc: Update docs for getbalance (default minconf should be 0)

### PR DESCRIPTION
**Pull request by vijaydasmp**
… be 0)

### CODE REVIEW

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -938,9 +938,9 @@ static UniValue getbalance(const JSONRPCRequest& request)
             "\nResult:\n"
             "amount              (numeric) The total amount in " + CURRENCY_UNIT + " received for this account.\n"
             "\nExamples:\n"
-            "\nThe total amount in the wallet with 1 or more confirmations\n"
+            "\nThe total amount in the wallet with 0 or more confirmations\n"
             + HelpExampleCli("getbalance", "") +
-            "\nThe total amount in the wallet at least 6 blocks confirmed\n"
+            "\nThe total amount in the wallet at least 6 blocks confirmation\n"
```
Comment (PastaPastaPasta) [inline src/wallet/rpcwallet.cpp:943]:
```suggestion
            "\nThe total amount in the wallet with at least 6 confirmations\n"
```

Comment (vijaydasmp) [reply src/wallet/rpcwallet.cpp]: Hello @PastaPastaPasta,  done the changes

**Review by PastaPastaPasta — DISMISSED** [review]
utACK for squash merge

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -938,9 +938,9 @@ static UniValue getbalance(const JSONRPCRequest& request)
             "\nResult:\n"
             "amount              (numeric) The total amount in " + CURRENCY_UNIT + " received for this account.\n"
             "\nExamples:\n"
-            "\nThe total amount in the wallet with 1 or more confirmations\n"
+            "\nThe total amount in the wallet with 0 or more confirmations\n"
             + HelpExampleCli("getbalance", "") +
-            "\nThe total amount in the wallet at least 6 blocks confirmed\n"
+            "\nThe total amount in the wallet with at least 6 confirmation\n"
```
Comment (UdjinM6) [inline src/wallet/rpcwallet.cpp:943]: still not fixed
```suggestion
            "\nThe total amount in the wallet with at least 6 confirmations\n"
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**